HTML/styles.scss

```diff
@@ -0,0 +1,53 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+
+    .level2 {
+        padding: 10px;
+        background-color: #d0d0d0;
+
+        .level3 {
+            padding: 10p;
+            background-color: #b0b0b0;
+
+            .level4 {
+                padding: 10px;
+                background-color: #909090;
+
+                .level5 {
+                    padding: 10px;
+                    background-color: #707070;
+                    color: white;
+                }
+            }
+        }
+    }
+}
+
+header nav ul{
+    width: 100%;
+    top: 45px;
+    
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+  
+    }
+  header nav ul li {
+    position: relative;
+  } 
+  header nav ul li:hover {
+    filter: brightness(8);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2208262726>#007008</a><span> - 07/04/2024, 07:11 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The brightness filter value of 8 is excessively high and can cause visual issues. Typically, brightness values range from 0 to 1.
</div>
<div id="fix">
<b>Fix:</b> Adjust the brightness filter value to a more reasonable level, such as 1.2 for a slight increase in brightness.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -40,7 +40,7 @@
 header nav ul li:hover {
 -    filter: brightness(8);
 +    filter: brightness(1.2);
```
</div>
</details>
</div>
<br>


</div>